config/event_handling.tmpl.yml

```diff
@@ -532,14 +534,6 @@ event_groups:
     - NODE_REBOOT
     - NODE_NODESCHEDULABLE
     - NODE_NODENOTSCHEDULABLE
-    - POD_SCHEDULED
-    - POD_FAILEDSCHEDULING
-    - REPLICATOR_SUCCESSFULCREATE
-    - REPLICATOR_FAILEDCREATE
-    - CONTAINER_CREATED
-    - CONTAINER_KILLING
-    - CONTAINER_STARTED
-    - CONTAINER_STOPPED
```

Comment:
I'm thinking 'Power Activity' should be used for power management related events. I've put 'CONTAINER_STARTED' and 'CONTAINER_STOPPED' under 'Alarm/Status change/Errors' while  'CONTAINER_CREATED' is in 'Creation/Addition'